runelite-client/src/test/java/net/runelite/client/util/ColorUtilTest.java

```diff
@@ -123,8 +126,66 @@ public void colorLerp()
 	public void colorToHexCode()
 	{
 		COLOR_HEXSTRING_MAP.forEach((color, hex) ->
-		{
-			assertEquals(hex, ColorUtil.colorToHexCode(color));
-		});
+			assertEquals(hex, ColorUtil.colorToHexCode(color)));
```

Comment:
While we don't have CI enforcing it in tests, we'd prefer it maintains the same code style as the sources, which requires a block for lambdas.

eg.

```java
map.forEach((k, v) ->
{
	// ...
});
```

rather than

```java
map.forEach((k, v) ->
	/* ... */ );
```

Reply:
I see. I had IntelliJ auto-suggest turning Lambdas with statement blocks into expressions. I'm using both the RuneLite CheckStyle and the IntelliJ formatter config, but it seems neither codifies this. 

Do you think this is something that should be added to CheckStyle? I could look into that.

Reply:
I wouldn't mind that being added in another PR, but that's best left separate from these changes.

Reply:
Sure, another time then.